src/state.tsx

```diff
@@ -438,6 +438,15 @@ class GlobalState implements State {
   }
 
   updateImportedData(data: ImportableData) {
+    /* eslint-disable no-fallthrough */
```

Comment:
Can we just add `break;` in these cases rather than disabling an eslint rule?

Reply:
The fallthrough here is desired behaviour, but it'll only become apparent whenever the next revision is added.

Once we're active version 3, loading a version 1 import should apply all the version 1->2 transformations under `case 1`, then also apply the 2->3 transformations under `case 2`. The import data's version number defines a start point for transforms, not the distinct set of transforms.